packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -118,7 +119,7 @@ export function PortableTextInput(props: PortableTextInputProps): ReactNode {
   const {t} = useTranslation()
   const [ignoreValidationError, setIgnoreValidationError] = useState(false)
   const [invalidValue, setInvalidValue] = useState<InvalidValue | null>(null)
-  const [isFullscreen, setIsFullscreen] = useState(false)
+  const [isFullscreen, setIsFullscreen] = useState(fullscreenProp ?? false)
```

Comment:
looking at this usage it seems like the `fullscreen` prop sets just the default value of this vs managing its state. In react, i typically see props that behave like this prefixed with `default*` e.g.e `defaultValue`. idk if `defaultFullscreen` would be a better or worse name but something to consider. not a blocking comment by any means.

Reply:
Great suggestion and makes perfect! Had a sniff around the studio and found other similar instances which use `initial*` so went with that approach